currencies/src/lib.rs

```diff
@@ -84,11 +85,13 @@ pub mod module {
 		type MultiCurrency: TransferAll<Self::AccountId>
 			+ MultiCurrencyExtended<Self::AccountId>
 			+ MultiLockableCurrency<Self::AccountId>
-			+ MultiReservableCurrency<Self::AccountId>;
+			+ MultiReservableCurrency<Self::AccountId>
+			+ NamedMultiReservableCurrency<Self::AccountId, ReserveIdentifier = [u8; 8]>;
```

Comment:
It's better to define a ReserveIdentifier type config here.

Reply:
There isn't a real need to define it. Just leave it out and use `ReserveIdentifierOf` when you need it.

`type ReserveIdentifierOf<T> = <<T as Config>::MultiCurrency as NamedMultiReservableCurrency <<T as frame_system::Config>::AccountId>>:: ReserveIdentifier;`